api/src/main/java/jakarta/persistence/criteria/CriteriaBuilder.java

```diff
@@ -1170,6 +1170,8 @@
      * if found.
      * The first position in a string is denoted by 1.  If the
      * string to be located is not found, 0 is returned.
+     * <p>Warning: the order of the parameters of this method is
```

Comment:
should the word _Warning_ be  bold/strong?

Reply:
Sure, if you like.